pkg/pillar/containerd/containerd.go

```diff
@@ -112,32 +110,23 @@ func containerdLoadImageTar(filename string) (map[string]images.Image, error) {
 	return names, nil
 }
 
-// CtrRm remove an existing container. If silent is true, then operation failures are ignored and no error is returned
-func CtrRm(containerPath string, silent bool) error {
-	log.Infof("ctrRm %s\n", containerPath)
+// SnapshotRm removes existing snapshot. If silent is true, then operation failures are ignored and no error is returned
+func SnapshotRm(containerPath string, silent bool) error {
+	log.Infof("snapshotRm %s\n", containerPath)
 
 	containerID := filepath.Base(containerPath)
-	container, err := loadContainer(containerID)
-	if err != nil {
-		err = fmt.Errorf("ctrRm: exception while loading container: %v. %v", containerID, err.Error())
+
+	if err := deleteBundle(containerID, silent); err != nil {
+		err = fmt.Errorf("snapshotRm: unable to delete bundle: %v. %v", containerID, err.Error())
 		log.Error(err)
 		if !silent {
 			return err
 		}
 	}
-	if container == nil {
-		return nil
-	}
-	if err := container.Delete(ctrdCtx, containerd.WithSnapshotCleanup); err != nil {
-		err = fmt.Errorf("ctrRm: unable to delete container: %v. %v", containerID, err.Error())
-		log.Error(err)
-		if !silent {
-			return err
-		}
 
-	}
-	if err := deleteBundle(containerID, silent); err != nil {
-		err = fmt.Errorf("ctrRm: unable to delete bundle of container: %v. %v", containerID, err.Error())
+	snapshotter := CtrdClient.SnapshotService(defaultSnapshotter)
```

Comment:
We should reference the official default snapshotter as:

```go
	snapshotter := CtrdClient.SnapshotService(containerd.DefaultSnapshotter)
```

Reply:
Why would we do that @deitch if we had introduced our own notion of what EVE wants the default snapshotter to be?

Reply:
All we are doing here is using the overlayfs, which is the default from containerd. Why rename it?